docs/getting-started/accessing-resources/getting-access-to-data.md

```diff
@@ -71,49 +59,71 @@ The [`download-data.py` script](https://github.com/AlexsLemonade/OpenScPCA-analy
 
     Note that you can also [add this profile definition to your shell profile file](../../technical-setup/environment-setup/configure-aws-cli.md#storing-your-aws-profile-name) so that it will always be defined in any terminal session.
 
-We briefly cover some of the most common use cases below, but we encourage you to review all the options available to you.
+- You can list all possible arguments and their usage for the download data script with the following:
 
-You can list all the options for the download data script by running the following from the root directory of the repository:
+    ```sh
+    ./download-data.py --help
+    ```
 
-```sh
-./download-data.py --help
-```
+- You can run the download script with all default options to download all processed samples from the most recent release in `SingleCellExperiment` format:
 
-You can run the download script with all default options to download all processed samples from the most recent release in `SingleCellExperiment` format:
+    !!! warning
+        **The full data release download is quite large: over 35 GB for `SingleCellExperiment` format, and over 100 GB for `AnnData` format.
+        Please download the full data release with caution!**
 
-!!! warning
-    **The full data release download is quite large: over 35 GB for `SingleCellExperiment` format, and over 100 GB for `AnnData` format.
-    Please download the full data release with caution!**
+    ```sh
+    ./download-data.py
+    ```
 
-```sh
-./download-data.py
-```
+- If you prefer to work with `AnnData` files, use the `--format` option as follows:
 
-If you prefer to work with `AnnData` files, use the `--format` option as follows:
+    ```sh
+    ./download-data.py --format AnnData
+    ```
 
-```sh
-./download-data.py --format AnnData
-```
+- To download a specific release other than the most recent release, you can use the `--release` option as follows.
```

Comment:
I don't feel like "as follows" is needed everywhere, as it becomes pretty self-evident.
```suggestion
- To download a specific release other than the most recent release, you can use the `--release` option.
```